### ISSUE TITLE
Merge users that are present in multiple cities

### ISSUE THREAD

**Issue by misaugstad** [ProjectSidewalk/sidewalk-quality-analysis#56]
For example, I have contributed data in every city, so we need to figure out how we want to deal with that.

**Comment by jonfroehlich** [comment]
Agree. I've been thinking about that as well. For now, this is a lower
priority than getting more feature columns to me. :)

And even an MVP take on this problem—something like making their user ids
the same across dumps, so I could do a post hoc merge—might be OK (yes, I
realize this won't be perfect because the 'post hoc' merge will be
averaging averages, etc. but could be OK for now).

Jon
